README.md

```diff
@@ -67,19 +69,6 @@ config.set({
         'test/index.js': 'webpack'
     },
     webpack: {
-        // *optional* babel options: isparta will use it as well as babel-loader
-        babel: {
-            presets: ['es2015', 'stage-0', 'react']
-        },
-        // *optional* isparta options: istanbul behind isparta will use it
-        isparta: {
-            embedSource: true,
-            noAutoWrap: true,
-            // these babel options will be passed only to isparta and not to babel-loader
-            babel: {
-                presets: ['es2015', 'stage-0', 'react']
-            }
-        },
         …
         module: {
             preLoaders: [
```

Comment:
preLoaders should be replaced with rules, for an accurate documentation using webpack 2